contrib/gcppubsub/cmd/receive_adapter/main.go

```diff
@@ -64,11 +58,17 @@ func main() {
 		log.Fatalf("Unable to create logger: %v", err)
 	}
 
+	var env envConfig
+	if err := envconfig.Process("", &env); err != nil {
+		log.Fatal("Failed to process env var", zap.Error(err))
+	}
+
 	adapter := &gcppubsub.Adapter{
-		ProjectID:      getRequiredEnv(envProject),
-		TopicID:        getRequiredEnv(envTopic),
-		SinkURI:        getRequiredEnv(envSinkURI),
-		SubscriptionID: getRequiredEnv(envSubscription),
+		ProjectID:      env.Project,
+		TopicID:        env.Topic,
+		SinkURI:        env.Sink,
+		SubscriptionID: env.Subscription,
+		TransformerURI: env.Transformer,
```

Comment:
Should we at least validate the URI to make sure it's a valid URI?

Reply:
we are not doing that for the sinkURI. I guess if it's not valid it will break when trying to create the httpClient. And it can be empty right? in that case we do not create any client...